### ISSUE TITLE
mongolab / mlab no longer exist

### ISSUE THREAD

**Issue by jcpeterson** [Tuuleh/jsPsychBackendStart#2]

**Comment by Tuuleh** [comment]
Yep. Thanks for the flag - this tutorial is due for a serious update but this one is definitely a major issue. I don't have a whole lot of free time to work on this ATM, especially since I don't work in behavioral research right now.
